bluesky_widgets/headless/figures.py

```diff
@@ -62,6 +62,12 @@ def _on_figure_removed(self, event):
         figure.close_figure()
         del self._figures[figure_spec.uuid]
 
+    def close_figures(self):
+        for figure in self._figures.values():
+            _close_figure(figure)
```

Comment:
The keys in this dict are `HeadlessFigure` instances, not `matplotlib.figure.Figure` instances, so this should be `figure.close_figure()`.